File: Bai_87/main.py
compressed = input()
result = []
i = 0
length = len(compressed)
while i < length:
    if compressed[i].isdigit():
        count = 0
        while i < length and compressed[i].isdigit():
            count = count * 10 + int(compressed[i])
            i += 1
        if i < length and compressed[i].isalpha():
            result.append(compressed[i] * count)
    else:
        result.append(compressed[i])
    i += 1
print(''.join(result))

File: Bai_87/main.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    string compressed;
    cin >> compressed;
    string res;
    for(int i = 0; i < compressed.size(); ++i){
        if(isdigit(compressed[i])) {
            int count = 0;
            while (i < compressed.size() && isdigit(compressed[i])) {
                count = count * 10 + (compressed[i] - '0');
                ++i;
            }
            if (i < compressed.size() && isalpha(compressed[i])) 
                res.append(count, compressed[i]);
        } 
        else res.push_back(compressed[i]);
    }
    cout << res;
    return 0;
}
